python_modules/libraries/dagster-k8s/dagster_k8s/job.py

```diff
@@ -635,6 +635,10 @@ def construct_dagster_k8s_job(
 
     resources = user_defined_resources if user_defined_resources else job_config.resources
 
+    user_defined_command = container_config.pop("command", None)
+    if not command:
+        command = user_defined_command
```

Comment:
`command` and `user_defined_k8s_config` args are not in the doc block for this method

probably be good to add them and explicitly call out the precedence order, maybe a comment here as well